task/06-conditions-n-loops-tasks.js

```diff
@@ -332,7 +389,23 @@ function isBracketsBalanced(str) {
  *
  */
 function timespanToHumanString(startDate, endDate) {
-    throw new Error('Not implemented');
+    let time = (endDate - startDate-0.1)/1000;
+    const min = 60;
+    const hour = min*60;
+    const day = hour*24;
+    const month = day*30;
+    const year = day*365;
+    return time<=45?'a few seconds ago':
```

Comment:
It would be much better to use "if () return" blocks from the point of view of code readability than with many ternary operators